Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -101,12 +101,20 @@ class OverviewViewController: UIViewController {
         super.viewWillAppear(animated)
         self.navigationController?.navigationBar.isHidden = true
         self.checkWhetherToPromptForSeedBackup()
+        self.updateMultiWalletBalance()
+        self.updateRecentActivity()
     }
     
     func initializeViews() {
         // Set a scroll listener delegate so we can update the nav bar page title text on user scroll.
         self.parentScrollView.delegate = self
+        self.parentScrollView.alwaysBounceVertical = true
```

Comment:
This can/should be done from the storyboard.

Reply:
resolved in `698f737 `